migrations/Version202101262300571286_taoDeliveryRdf.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoDeliveryRdf\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\oatbox\event\EventManager;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoDeliveryRdf\model\DataStore\DeliveryMetadataListener;
+use oat\taoDeliveryRdf\model\event\DeliveryCreatedEvent;
+
+final class Version202101262300571286_taoDeliveryRdf extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Registering DataStoreService for metaData processing';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $eventManager = $this->getEventManger();
```

Comment:
We need to register event manager with new events attached

Reply:
done , forgot , thank you